README.md

```diff
@@ -195,7 +195,9 @@ for IaaS settings.
   another BOSH Director, you will be prompted for the name of that BOSH director.
   This name should match the alias you have configured in your BOSH config, as it
   will tell Genesis how to communicate with that BOSH Director to deploy BOSH.
-
+- params.session_timeout - The default BOSH login session timeout is 1 day. You can 
+  overwrite it in your environment yaml file if you would like a different session 
+  timeout value.
```

Comment:
This should follow the same markdown formatting as above (the param name should be surrounded by `**`)

Override, not overwrite.

Lead off the description with what session timeouts do, why an operator would care, etc.  End with the default.  Something like:

> When users authenticate to BOSH, they will be given a limited-lifetime session during which they do not have to re-authenticate.  Once that session expires, however, they will need to provide their credentials again.  This parameter let's you set how long that session lasts.  Defaults to 1 day.

Also need to be clear on units.  Is it seconds?  milliseconds?  days?  Does "30d" work?